src/api/wishcards.ts

```diff
@@ -1,8 +1,21 @@
 import express from 'express';
 
-// import WishCardController from '../controller/wishcard';
+import Permissions from '../middleware/permissions';
+import Validations from '../middleware/validations';
+
+import WishCardApiController from './controller/wishcards';
 
 const router = express.Router();
-// const wishCardController = new WishCardController();
+const wishCardController = new WishCardApiController();
+
+router.get('/agency_me', Permissions.isAdminOrAgency, wishCardController.getAgencyWishcards);
+
+router.put(
+	'/agency_me',
+	Permissions.isAdminOrAgency,
+	Validations.updateAgencyWishcardValidationRules(),
+	Validations.validate,
+	wishCardController.putAgencyWishCardById,
+);
```

Comment:
we should consider renaming this, <something>_me is not clear what it means
If it's related to wish cards, then name it accordingly

Reply:
so should '/api/wishcards/agency' works?

Reply:
Yeah I agree that "me" is not very clear. How about  /wishcard/manage? It's more consistent with the action button that says "manage wish cards" too.

Reply:
so are you guys ok with '/wishcards/manage'  for the manage wishcards page route for partner user and '/api/wishcards/agency/manage' for the api route ? @stacysealky @Enubia

Reply:
Yes that would be ok